Makefile

```diff
@@ -31,10 +31,12 @@ tests/%:
 	$(MAKE) -C tests $*
 
 .PHONY: generated_files
+ifneq ($(GEN_FILES), 0)
```

Comment:
It's a shame that CMake uses `GEN_FILES = ON` but `Makefile` uses `0` - can they not be the same?

Reply:
[Cmake recognizes multiple strings as booleans](https://cmake.org/cmake/help/latest/command/if.html#constant), so `GEN_FILES=0` works with cmake too.